src/dotnet/Fable.JS/Main.fs

```diff
@@ -195,14 +206,15 @@ let defaultManager =
         member __.GetCompletionsAtLocation(parseResults:IParseResults, line:int, col:int, lineText:string) =
             let res = parseResults :?> ParseResults
             getCompletionsAtLocation res line col lineText
-        member __.CompileToBabelAst(com: IFableCompiler, parseResults:IParseResults, fileName:string, optimized: bool) =
-            let com = com :?> CompilerImpl
+        member __.CompileToBabelAst(fableCore:string, parseResults:IParseResults, fileName:string, optimized: bool) =
```

Comment:
Ah! I realize now that you actually wanted to pass fableCore from the API not from the JS config file, sorry! Well, this makes sense, but then it may be confusing to have two ways to manage the fable-core path (using a global variable or passing it as a local parameter) probably we should remove the global variable for simplicity....

Reply:
I agree.